app/helpers/application_helper.rb

```diff
@@ -231,7 +231,7 @@ def format_validation_error_messages(obj, options = {})
 
   # pill label
   def pill_label(text, kind: "default")
-    content_tag(:span, text, class: "label label-#{kind}")
+    content_tag(:span, text, class: "badge badge-pill badge-#{kind}")
```

Comment:
Let's take this out. I don't really like it anymore anyway. Let's pair on what to do with it.